db/Schema.dbml

```diff
@@ -155,31 +147,29 @@ enum voyage_status {
 
 
 // A user has 1 role
-ref: user_roles.id < users.fk_role_id
+ref: user_roles.id < users.role_id
 
 // A company has 0-* users
-ref: organizations.id < users.fk_company_id
+ref: organizations.id < users.organization_id
 // A vessel has 0-* voyages
-ref: vessels.id < voyages.fk_id_vessel
-// A voyage has 0-* routes
-ref: voyages.id < suggested_routes.fk_id_voyage
+ref: vessels.id < voyages.vessel_id
+
+ref: vessel_reports.id < suggested_routes.vessel_report_id
+// A suggested_routes has 0-* vessel_reports
+ref: suggested_routes.id < vessel_reports.selected_route_id
 // A route has 1-* waypoints
-ref: suggested_routes.id < waypoints.fk_suggested_route_id
+ref: suggested_routes.id < waypoints.suggested_route_id
 // A port has 1-* waypoints
-ref: ports.id < waypoints.fk_port_id
+ref: ports.id < waypoints.port_id
 // A company has 0-* vessels
-ref: organizations.id < vessels.fk_id_company
+ref: organizations.id < vessels.organization_id
 // A vessel has 0-* vessel_reports
-ref: vessels.id < vessel_reports.fk_vessel_id
+ref: vessels.id < vessel_reports.vessel_id
 // A voyage has 1 departure waypoint
 ref: voyages.depart_from > waypoints.id
 // A voyage has 1 arrival waypoint
 ref: voyages.arrive_at > waypoints.id
 // A waypoint may belong to 0-* vessel reports
-ref: waypoints.id < vessel_reports.position 
-// Selected route represents a many-to-many join table between route and voyage
-// Current selected route is found by looking for the latest record
-ref: suggested_routes.id < selected_routes.fk_route_id
-ref: voyages.id < selected_routes.fk_voyage_id
-ref: favorite_vessels.user_id > users.id
-ref: favorite_vessels.vessel_id > vessels.id
```

Comment:
You removed the references between favorite_vessels, users and vessels? Is that because we don't have them refenced in the DB or is it an accident?